cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,60 @@
+import { faker } from '@faker-js/faker';
+import HomeAndCataloguePageObject from '../support/pages/homeСatalogue.pageObject';
+import CartPageObject from '../support/pages/cart.pageObject';
 /// <reference types='cypress' />
 
-describe('', () => {
-  before(() => {
+const homePage = new HomeAndCataloguePageObject();
+const cartPage = new CartPageObject();
+
+const testData = {
+  name: faker.name.firstName(),
+  country: faker.address.country(),
+  city: faker.address.city(),
+  creditCard: faker.finance.creditCardNumber(),
+  month: faker.date.month(),
+  year: faker.date.future({ years: 5 }).getFullYear(),
```

Comment:
The method `faker.date.future({ years: 5 }).getFullYear()` might not work as expected because `faker.date.future()` returns a Date object, and calling `getFullYear()` directly on it may not be valid. Consider using `faker.date.future(5).getFullYear()` instead.